website/src/components/expandable/styles.module.css

```diff
@@ -0,0 +1,86 @@
+:local(.link) :local(.headerText) {
+    content: '';
+    color: black; /* Black text in normal mode */
+    text-decoration: none;
+    transition: text-decoration 0.3s; /* Smooth transition */
+    font-weight: 600;
+    margin-bottom: 20px;
+}
+
+:local(.link:hover) :local(.headerText),
+:local(.link:focus) :local(.headerText) {
+    text-decoration: underline;
+    cursor: pointer;
+}
+
+:local(.toggle)::before {
+    content: '';
+    display: inline-block;
+    border: solid black; /* Arrow color */
+    border-width: 0 2px 2px 0; /* Adjust for 'boldness' */
+    padding: 3px; /* Adjust size */
+    transform: rotate(45deg); /* Initial right-pointing arrow */
+    transition: transform 0.3s; /* Smooth transition for toggle icon */
+    margin-right: 8px;
+}
+
+:local(.toggleDown)::before {
+    transform: rotate(225deg); /* Downward arrow */
+}
+
+:local(.toggleRight)::before {
+    transform: rotate(315deg); /* Right-pointing arrow */
+}
+
+:local(.toggle)::before {
+    border-color: rgb(253, 153, 83); /* Orange arrow color in light mode */
+}
+
+/* Adjusting for Light and Dark Modes */
+:local(html[data-theme='dark'] .link), :local(html[data-theme='dark'] .headerText) {
+    color: white; /* White text in dark mode */
+}
+
+:local(html[data-theme='dark'] .toggle)::before {
+    border-color: white; /* White arrow in dark mode */
+    border-color: rgb(253, 153, 83);
+}
+
+:local(.body) {
+    margin-top: 10px;
+    margin-left: 1em;
+    padding: 10px;
+    background-color: transparent; /* Changed from a color to transparent */
+}
```

Comment:
The content within the toggle aligns slightly more inward than the `alt_header` text. Can we adjust the styles to align the content with the header, similar to stripe's docs?

Also, just a heads up if we change any styles here, they will affect all elements on the page which have these classes. That means if the FAQs component is used on the same page as this expandable component, there might be conflicting styles.

It would be best to scope these styles specifically to this expandable component. Since the `expandableContainer` class wraps all other elements. The easiest way to scope the styles to this component is by prefixing the classes in this css with `.expandableContainer`, for example: `.expandableContainer :local(.body)`. I'd recommend doing this for all css declarations in this file.

![image](https://github.com/dbt-labs/docs.getdbt.com/assets/13647110/edd45070-d7cf-49fe-a650-1ab716bfbb29)

Reply:
amazing, thank you so much for the guidance @JKarlavige !! ![Screenshot 2024-02-09 at 12 46 45](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/0b871e57-3eb1-4fc7-aca8-ddb5a4b2d35b)

Reply:
thank you so much @JKarlavige ! made that change! 
![Screenshot 2024-02-09 at 12 57 18](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/5adb5cc0-8ff2-477f-b9ca-33c391313516)